src/coreclr/nativeaot/Runtime/eventtrace.cpp

```diff
@@ -3588,21 +3588,24 @@ INT32 WideCharToMultiByte(LPCWSTR wszSrcStr, LPSTR szDstStr);
 
 ULONG ETW::CEtwTracer::Register()
 {
-    // Get Env Var COMPlus_ETWEnabled
-    char szETWEnabled[SIZE_ETWEnabled];
-    DWORD newLen = GetEnvironmentVariableA("COMPlus_ETWEnabled", szETWEnabled, SIZE_ETWEnabled);
+    // Get Env Var DOTNET_ETWEnabled or COMPlus_ETWEnabled
+    char* szETWEnabled[SIZE_ETWEnabled] = { '\0' };
```

Comment:
```suggestion
    char szETWEnabled[SIZE_ETWEnabled] = { '\0' };
```

Seems wrong to have added a `*` to the type

Reply:
I wonder if this code compiles at all?